app/model/ReviewData.scala

```diff
@@ -0,0 +1,3 @@
+package model
+
+case class ReviewData(confirmRecordTransfer: String, confirmOpen: String, confirmTnaOwnership: String)
```

Comment:
I've copied the format of the form data from the service agreement page, but I think these fields should be booleans rather than Strings. Is there a reason that they're not booleans? @TomJKing - do you know?

Reply:
They should be booleans.

Reply:
As discussed in person, these should be booleans but we're going to leave them as they are in the prototype and fix them if we have time or if makes sense when implementing something else.